src/Mutations/ImportModels.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mrap\GraphCool\Mutations;
+
+use GraphQL\Type\Definition\ResolveInfo;
+use Mrap\GraphCool\DataSource\DB;
+use Mrap\GraphCool\DataSource\File;
+use Mrap\GraphCool\DataSource\FullTextIndex;
+use Mrap\GraphCool\Definition\ModelBased;
+use Mrap\GraphCool\Definition\Mutation;
+use Mrap\GraphCool\Types\Type;
+use Mrap\GraphCool\Utils\Authorization;
+use Mrap\GraphCool\Utils\JwtAuthentication;
+use RuntimeException;
+use function Mrap\GraphCool\pluralize;
+
+class ImportModels extends Mutation
+{
+    use ModelBased;
+
+    public function __construct(?string $model = null)
+    {
+        if ($model === null) {
+            throw new RuntimeException(__METHOD__.': parameter $model may not be null for ModelBased mutations.');
+        }
+        $plural = pluralize($model);
+        $this->name = 'import'.$plural;
+        $this->model = $model;
+
+        $this->config = [
+            'type' => Type::get('_ImportSummary'),
+            'description' => 'Import a list of ' . $plural . ' from a spreadsheet. If ID\'s are present, ' . $plural . ' will be updated - otherwise new ' . $plural . ' will be created. To completely replace the existing data set, delete everything before importing.',
+            'args' => $this->importArgs($model)
+        ];
+    }
+
+    public function resolve(array $rootValue, array $args, mixed $context, ResolveInfo $info): mixed
```

Comment:
Rückgabewert ist `mixed`? Sollte man da nicht `object` angeben?

Reply:
graphql-php verwendet an und für sich mixed für alle resolver.

In meinen eigenen resolvern könnte ich das theoretisch anders machen... aber ich kann nicht einheitlich überall object oder stdClass machen, weil manchmal dann doch ein String oder Int oder Bool benötigt wird...